HISTORY.md

```diff
@@ -1,5 +1,46 @@
 # Release History - `open-autonomy`
 
+# 0.2.0 (2022-08-09)
+
+# AEA
+ 
+- Adds support for registry flags on `autonomy scaffold fsm` command
+- Adds support for scaffolding
+  - Dialogues
+  - Payloads
+  - Round tests
+  - Behaviour tests
```

Comment:
also scaffolding tests for: handlers, models, payloads, dialogues

Reply:
Just `Tests` then?